### PR TITLE
Share Gatling tests and performences scriptes

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ install:
 - .build/venv/bin/flake8 --version
 
 script:
-- if [ ${TRAVIS_PULL_REQUEST}} != "false" ] ; then git diff --check ${TRAVIS_BRANCH} ; fi
+- if [ ${TRAVIS_PULL_REQUEST}} != "false" ] ; then git diff --check ${TRAVIS_BRANCH} -- ; fi
```
Comment (ochriste) [inline .travis.yml]: what is this for?

Comment (sbrunner) [inline .travis.yml]: Have a look on the comment ?  https://github.com/camptocamp/c2cgeoportal/commit/b2fbd10fdac919c880b79d51f27b8bf73ca8df3a
(the branch has the same name as a folder ...)

Comment (ochriste) [inline .travis.yml]: thx

**Comment by ochriste** [comment]
LGTM

**Pull request merged**